config/history.go

```diff
@@ -80,21 +80,20 @@ func (h *History) RecordForRuntimeRound(runtime common.Runtime, round uint64) (*
 	)
 }
 
-func SingleRecordHistory(chainContext string) *History {
-	return &History{
-		Records: []*Record{
-			{
-				ArchiveName:   "damask",
-				GenesisHeight: 1,
-				ChainContext:  chainContext,
-			},
-		},
-	}
-}
-
 var DefaultChains = map[common.ChainName]*History{
 	common.ChainNameMainnet: {
 		Records: []*Record{
+			{
+				// https://github.com/oasisprotocol/mainnet-artifacts/releases/tag/2023-11-29
+				ArchiveName:   "eden",
```

Comment:
Just FYI, this archive name is referenced [here](https://github.com/oasisprotocol/nexus/blob/6b2fe2e4d580e7f9be194081684c19dcf7f1a6bf/storage/oasis/nodeapi/history/history.go#L50-L50) to tell nexus what API to use to talk to the node. (The 23.0.x API is called "eden" internally.) Should be preconfigured correctly, but hasn't been tested, obviously.